SAMD21/RS485_PRESSURE/RS485_PRESSURE.ino

```diff
@@ -9,16 +9,16 @@ const int numChannels = 8;
 String address = "pres";
 const String comma = ",";
 const String end_mark = "end";
-evolver_si in("pres", "_!", numChannels + 1);
+evolver_si in("pres", "_!", numChannels + 1); // command string from eVOLVER
 
-float V_SOURCE = 3.3;
+float V_SOURCE = 3.3; // Voltage coming from the Arduino
```

Comment:
This is not supplied by the arduino - there is an LDO on the ADC board that supplies it. It's used to convert the ADC to a pressure based on the spec of the guage on line 177: https://media.digikey.com/pdf/Data%20Sheets/Honeywell%20PDFs/HSC_DS.pdf